src/lib/dacc.spec.js

```diff
@@ -0,0 +1,275 @@
+import { createMockClient, models } from 'cozy-client'
+const { sendMeasureToDACC } = models.dacc
+
+import * as dacc from './dacc'
+jest.mock('cozy-client', () => ({
+  ...jest.requireActual('cozy-client'),
+  models: {
+    dacc: {
+      sendMeasureToDACC: jest.fn()
+    }
+  }
+}))
+jest.mock('cozy-flags')
+
+const mockedCurrentDate = '2022-04-12T11:01:58.135Z'
+
+jest.mock('date-fns/is_this_month', () => date => {
```

Comment:
nit: si ça peut simplifier, de mon côté j'ai installé https://www.npmjs.com/package/mockdate

Reply:
Good to know, pour la prochaine fois :+1: